src/fillTank.js

```diff
@@ -21,11 +21,13 @@ function fillTank(customer, fuelPrice, amount = Infinity) {
   const roundedAmount = roundFuel(requiredAmount);
 
   if (roundedAmount < 2) {
```

Comment:
Consider adding a check to ensure that `roundedAmount` is not negative. Although the current logic seems to prevent this, it's a good practice to explicitly handle such cases to avoid potential issues.